vendor.conf

```diff
@@ -27,8 +27,8 @@ github.com/imdario/mergo v0.3.6
 golang.org/x/sync 1d60e4601c6fd243af51cc01ddf169918a5407ca
 
 # buildkit
-github.com/moby/buildkit c35410878ab9070498c66f6c67d3e8bc3b92241f
-github.com/tonistiigi/fsutil 1ec1983587cde7e8ac2978e354ff5360af622464
+github.com/moby/buildkit e9aca5bef87e19173b99d8668db0338dcaaa5f33
```

Comment:
do we want to release BuildKit 0.5 with this?

Reply:
@AkihiroSuda lets try to get some more things from https://github.com/moby/buildkit/milestone/2 in before doing that.